packages/framework/fluid-static/src/types.ts

```diff
@@ -215,3 +216,8 @@ export interface IMember {
      */
     connections: IConnection[];
 }
+
+/**
+ * An extended member object that includes currentConnectionId
+ */
+export type IMyself<M extends IMember = IMember> = M & { currentConnectionId: string; };
```

Comment:
why are we adding this new type?

Reply:
We want a new type so that there isn't any confusion between Myself and Member. Myself is more of an extension Member as it only represents one connection. Also, in the future, we may want to include fields such as "additionalDetails" which is specific to a connection rather than a Member.